templates/docker-compose.yml

```diff
@@ -72,6 +72,9 @@ services:
       - {{ key }}={{ value }}
 {% endfor %}
 {% endfor %}
+{% endif %}
+{% if pictrs_safety is defined and pictrs_safety|bool == true %}
+      - PICTRS__MEDIA__EXTERNAL_VALIDATION=http://{{ domain }}:14051/api/v1/scan/IPADDR
```

Comment:
It is worth considering proxying this via nginx instead of directly. This has the added benefit of being able to use `https` easily.

Reply:
There is no reason to open a public port and go through the public domain for this. It should work just fine to use the internal Docker network, by setting the url `http://pictrs-safety:14051/api/v1/scan/IPADDR`. Then the port below doesnt need to be exposed.

Another matter is that some admins may want to run pictrs-safety on a separate server with GPU, or run an entirely different verification software. With that in mind I would consider only adding a new parameter `pictrs_validation_url` and linking instructions to install pictrs-safety manually.

Reply:
About the ports, if pictrs-safety is installed, you likely **do** want the external connection to allow GPU workers running fedi-safety to connect and pick up validation jobs.

Reply:
Ah so the pictrs-safety image below doesnt do the validation on its own, but only hands out validation jobs to external workers? Then this setup makes sense. Anyway you should also include a documentation link which explains how to set this all up. You could write a page about this in the lemmy docs (or somewhere else).

Reply:
the pictrs-safety repo readme does provide full documentation, including a docker-compose sample
The fedi-safety likewise provides full documentation for connecting to pictrs-safety and also a docker container if someone wants to use it that way.

If this is accepted into the ansible repo, I'll work on adding doc to lemmy docs as well